habitat-baselines/habitat_baselines/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import pickle as pkl
+from typing import List
+
+import magnum as mn
+import numpy as np
+
+
+class Pose:
```

Comment:
More docstrings on these classes. This is the core of the PR, make sure these classes are understood correctly

Reply:
added